src/GovUk.Education.ExploreEducationStatistics.Common/Extensions/HttpRequestExtensions.cs

```diff
@@ -114,4 +114,13 @@ public static bool GetRequestParamBool(
         var paramValue = GetRequestParam(httpRequest, paramName, defaultValue + "");
         return bool.Parse(paramValue);
     }
+
+    public static int GetRequestParamInt(
+        this HttpRequest httpRequest,
+        string paramName,
+        int defaultValue)
+    {
+        var paramValue = GetRequestParam(httpRequest, paramName, defaultValue + "");
```

Comment:
Could just do `defaultValue.ToString()`?

Reply:
Done

Reply:
And for the bool equivalent also.